### ISSUE TITLE
Accessibility:  Consider making paper-icon tag an `img` tag vs.  `md-icon`

### ISSUE THREAD

**Issue by ssirowy** [miguelcobain/ember-paper#276]
We are considering using Paper in an educational app where accessibility is of high importance for our users. For the most part, this library cooperates with screen readers we've tested with.

We'd like to make use of many of the paper icons in our app. We've used several directly in buttons for image-like buttons. But since Paper icons aren't actually images, a screen reader has nothing to read (mainly the `alt` tag) to gather any context on the button's importance. 

It would be great if:
* paper icon's base tag was `img`
* The component bound on an `alt` property to allow clients to define a nice description for the button's utility.

**Comment by mike1o1** [comment]
I'm not very familiar with accessibility, but would it make more sense to use `aria-tags` for this, instead of changing the tag name? We can use an `aria-label` with the icon name - would that suffice?

In addition, if you're using an icon as a button, it should be wrapped in a `{{paper-button}}`, so wouldn't that get picked up by the screen reader?

**Comment by ssirowy** [comment]
The button is picked up by the screen reader, but with paper icon inside, currently there is nothing to report.  The screen reader will just say "BUTTON" which isnt particularly helpful.  

Support `aria-label` on either the paper icon, paper button, or both and be configurable by client would work.

**Comment by miguelcobain** [comment]
Note: angular material adds `aria-label="icon-name"` to icons. Shouldn't be difficult to do this ourselves.

**Comment by ssirowy** [comment]
Bonus points for making the label configurable by client.

E.g
<img width="165" alt="screen shot 2016-02-03 at 6 16 46 pm" 
src="https://cloud.githubusercontent.com/assets/191263/12803819/5aaf0a42-caa2-11e5-9645-a48842ef4195.png">

Getting icon name for an icon like this is more or less useless from an accessibility standpoint.

**Comment by miguelcobain** [comment]
@ssirowy You'd rather use something like `{{paper-icon "add-circle-online" aria-label="something here"}}`?

**Comment by ssirowy** [comment]
Yes I would.  I think the sensible default for `aria-label` would then be the icon's name as some icons do have some contextual information.

**Comment by miguelcobain** [comment]
This is just a matter of having an attributeBinding to the icon class. It's like 2 lines of code.

**Comment by ssirowy** [comment]
Yep. :+1:
